conf/defaults.ini

```diff
@@ -985,6 +985,13 @@ enabled = true
 # Enable the Profile section
 enabled = true
 
+#################################### News #############################
+[news]
+# Enable the news feed section
+news_feed_enabled = false
+# Feed used in the news drawer. Default to https://grafana.com/blog/news.xml
+news_feed_url = 
```

Comment:
if i comment this out/leave it blank in my `custom.ini`, nothing shows in the feed. feels like the default isn't being set correctly?

Reply:
alright, i will check.

Reply:
Fixed. Thanks Ash for checking it